docs/codeql/codeql-for-visual-studio-code/customizing-settings.rst

```diff
@@ -55,17 +57,58 @@ By default, items in the query history view are retained for 30 days. You can se
 
 .. _configuring-settings-for-running-queries:
 
-Configuring settings for running queries
------------------------------------------
+Configuring settings for running queries locally
+------------------------------------------------
 
 There are a number of settings for **Running Queries**. If your queries run too slowly and time out frequently, you may want to increase the memory. 
 
 .. include:: ../reusables/running-queries-debug.rst
 
 To save query server logs in a custom location, edit the **Running Queries: Custom Log Directory** setting. If you use a custom log directory, the extension saves the logs permanently, instead of deleting them automatically after each workspace session. This is useful if you want to investigate these logs to improve the performance of your queries.
 
-Configuring settings for testing queries
------------------------------------------
+Configuring settings for variant analysis
+------------------------------------------
+
+You can define or edit lists of GitHub repositories for variant analysis, and change to a different controller repository using the **Variant analysis** settings.
+
+For information on the purpose and requirements for a controller repository, see ":ref:`Setting up a controller repository for variant analysis <controller-repository>`."
```

Comment:
Will this link work? 🤔 (sorry, not familiar with RST stuff, but shouldn't you be linking to the containing article?). Feel free to ignore this comment if it doesn't apply 😅

Reply:
Thanks for checking in on this. I can confirm that the link works. Sphinx allows you to create named anchors that you can reference using this `ref` terminology. The reason I used this was that the link will continue to work even if the section heading changes.